about/policy/license-disclaimer/index.md

```diff
@@ -24,18 +29,38 @@ The data and services provided by UGRC is licensed under a [Creative Commons Att
 ### License Attribution
 {: .text-left}
 
-UGRC requests that where possible and practical, **appropriate attribution** shall be made by users of this website, data, or any other services provided by UGRC with the following criteria:
+UGRC requests that wherever possible and practical, you include **appropriate attribution** when using data or services provided by UGRC in your maps, web applications, or other content. This includes providing credit to UGRC and indicating you made any changes to the data.
```

Comment:
nice work on these examples.  i guess the BY is what's making our license a little more complex.

Reply:
I think our language of wherever possible and practical loosens it up. We could list that we are open to exceptions?